core/src/cell.rs

```diff
@@ -31,22 +30,24 @@ impl BlockInfo {
 #[derive(Clone, Eq, PartialEq, Default, Deserialize, Serialize)]
 pub struct CellMeta {
     #[serde(skip)]
-    pub cell_output: Option<CellOutput>,
+    pub cell_output: CellOutput,
```

Comment:
Why remove the option, should be cell_output or block_info

Reply:
No, it's not the original purpose, `block_info` only represents metadata of a CellOutput, a CellMeta is represented to a CellOutput with metadata.

The reason to use Option before is because sometimes we only need metadata, use Option can avoid loading CellOutput from DB every time, but it's confusing, and in the current design, we remove data from CellOutput, so load CellOutput become affordable, so I remove the Option.

Reply:
I see.